packages/peregrine/lib/store/reducers/user.js

```diff
@@ -17,23 +17,23 @@ const initialState = {
         firstname: '',
         lastname: ''
     },
-    getDetailsError: {},
-    isGettingDetails: false,
+    createAccountError: null,
+    getDetailsError: null,
+    isCreatingAccount: null,
+    isGettingDetails: null,
+    isResettingPassword: null,
```

Comment:
They are _sort of_ view states. What if you have multiple unrelated views that need to know if you are resetting the password? We started pulling out loading/wait and error states but decided to leave them in the global state because of all the coupled views.